src/timepicker/timepicker.js

```diff
@@ -132,15 +62,18 @@ class TimePicker extends React.Component<TimePickerPropsT, TimePickerStateT> {
         steps: steps,
         value: this.props.nullable
           ? undefined
-          : {id: closestStep, label: secondsToLabel(closestStep)},
+          : {
+              id: closestStep,
+              label: this.secondsToLabel(closestStep, undefined),
```

Comment:
```suggestion
              label: this.secondsToLabel(closestStep, this.props.format),
```

Reply:
@chasestarr mentioned we should keep this to maintain the previous behavior